daphne_worker/src/config.rs

```diff
@@ -151,15 +157,16 @@ impl DaphneWorkerConfig {
             None
         };
 
-        let collect_id_key = if is_leader {
-            let collect_id_key_hex = env
-                .secret("DAP_COLLECT_ID_KEY")
-                .map_err(|e| format!("failed to load DAP_COLLECT_ID_KEY: {e}"))?
+        const DAP_COLLECTION_JOB_ID_KEY: &str = "DAP_COLLECTION_JOB_ID_KEY";
```

Comment:
Reviewer note: This environment variable is OK to change right now as we don't have the Leader deployed anywhere.